ts/test/test.googleauth.ts

```diff
@@ -497,7 +500,7 @@ describe('GoogleAuth', () => {
 
     it('should error on non-string file path', (done) => {
       const auth = new GoogleAuth();
-      auth._getApplicationCredentialsFromFilePath(2, (err) => {
+      auth._getApplicationCredentialsFromFilePath(2 as any, (err: Error) => {
```

Comment:
This is kind of funny.  The test was verifying the type of the first argument.  I added typing to the method, and now it won't compile if you try to pass a Number. The test is in TypeScript as well, so it's impossible to pass the wrong type unless you cast to <any>.